myapp/app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new
+  end
```

Comment:
any plan to use this function? or we can remove it too?

Reply:
Actually it is indeed in used to render `new` view